src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -125,6 +126,10 @@ import ListPeopleAndRoles from './ListPeopleAndRoles.vue'
   }
 })
 export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
+  // Global state
+  @State(state => state.stateModel.addPeopleAndRoleStep.valid)
```

Comment:
Please use a getter instead. The components should know as little as possible about the store structure. See other components for examples.